rel/overlay/etc/default.ini

```diff
@@ -279,6 +279,8 @@ iterations = 10 ; iterations for password hashing
 ; min_iterations = 1
 ; max_iterations = 1000000000
 ; password_scheme = pbkdf2
+; List of tuples of an Erlang RegExp and an optional error message. Where a new password must match all RegExp. Example: [{".{10,}", "Password min length is 10 characters."}]
```

Comment:
Alternatively it could encourage folks to provide proper Reasons, which is likely beneficial down the road :) — I wouldn’t worry about shortcuts here.